library/src/main/java/com/github/paolorotolo/appintro/AppIntroViewPager.java

```diff
@@ -121,7 +121,7 @@ private boolean checkPagingState(MotionEvent event) {
             return true;
         }
 
-        if (!nextPagingEnabled) {
+        if (nextPagingEnabled) {
```

Comment:
I take it this 'fixes' being able to scroll back when making use of `isPolicyRespected` -- but what are the side-effects of changing this eval.
Have you tested swiping without making use of `isPolicyRespected`?